db/migrate/20150106050733_set_mysql_to_unicode_mb4.rb

```diff
@@ -0,0 +1,38 @@
+class SetMysqlToUnicodeMb4 < ActiveRecord::Migration
+
+  UTF8_PAIRS = {
+    'comments': 'text',
+    'messages': 'text',
+    'poll_answers': 'answer',
+    'polls': 'question',
+    'posts': 'text',
+  }
+
+  def self.up
+    if ENV['DB'] == 'mysql'
```

Comment:
I plan to eventually get rid of the DB environment variable, I think you should be able to query the adapter through AR::Base.config or something like that.